code/game/gamemodes/wizard/wizard.dm

```diff
@@ -9,162 +9,33 @@
 	required_players = 20
 	required_enemies = 1
 	recommended_enemies = 1
-	enemy_minimum_age = 14
+	enemy_minimum_age = 0

```

Comment:
I disagree. I remember seeing a newbie get wizard before the minimum age was added, and the round ended incredibly quickly. Not giving people time to learn the game results in boring rounds and also interrupts their actual learning of the game, especially with a role so intrusive as wizard.